packages/@webex/plugin-meetings/test/unit/spec/breakouts/index.ts

```diff
@@ -289,5 +292,78 @@ describe('plugin-meetings', () => {
         assert.equal(breakouts.isInMainSession, true);
       });
     });
+
+    describe('#breakoutServiceUrlUpdate', () => {
+      it('sets the breakoutService url', () => {
+        breakouts.breakoutServiceUrlUpdate('newBreakoutServiceUrl');
+        assert.equal(breakouts.breakoutServiceUrl, 'newBreakoutServiceUrl/breakout/');
+      });
+    });
+
+    describe('#toggleBreakout', () => {
+      it('enableBreakoutSession is undefined, run enableBreakouts then toggleBreakout', async() => {
+        breakouts.enableBreakoutSession = undefined;
+        breakouts.enableBreakouts = sinon.stub().returns(Promise.resolve({body: {
```

Comment:
minor thing, but `.returns(Promise.resolve(...))` can be replaced with `.resolves(...)`